src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImplTest.java

```diff
@@ -42,12 +42,15 @@ void processMessageTest() throws ServiceException {
         messageProcessorService.processMessage(chipsRestInterfacesSend);
 
         verify(chipsRestClient, times(1)).sendToChips(eq(chipsRestInterfacesSend));
-        verify(retryMessageProducer, times(0)).writeToTopic(eq(chipsRestInterfacesSend));
+        verify(messageProducer, times(0)).writeToTopic(eq(chipsRestInterfacesSend), eq("chips-rest-interfaces-send-retry"));
+        verify(messageProducer, times(0)).writeToTopic(eq(chipsRestInterfacesSend), eq("chips-rest-interfaces-send-error"));
     }
 
     @Test
     void testRetryIsCalled() throws ServiceException {
         ReflectionTestUtils.setField(messageProcessorService, "maxRetryAttempts", 10);
+        ReflectionTestUtils.setField(messageProcessorService, "retryTopicName", "chips-rest-interfaces-send-retry");
+        ReflectionTestUtils.setField(messageProcessorService, "errorTopicName", "chips-rest-interfaces-send-error");
```

Comment:
the topic name is duplicated lots of times, you can make it a class constant

Reply:
Refactored